### PR TITLE
Add Shoreline integration tile

### PR DESCRIPTION
**Pull request by ytnytn1**
### What does this PR do?

Adds Shoreline tile

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: shoreline/manifest.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      {
+        "media_type": "image",
+        "caption": "Example of fleetwide linux command details",
+        "image_url": "images/fleetwide_linux_command_details.png"
+      }
+    ]
+  },
+  "author": {
+    "homepage": "https://shoreline.io/",
+    "vendor_id": "TODO Please Fill In",
```
Comment (mgarabed) [inline shoreline/manifest.json:45]:
```suggestion
```

File: shoreline/manifest.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        "metadata_path": "assets/service_checks.json"
+      }
+    }
+  },
+  "legal_terms": {
+    "eula": "assets/terms_of_service_shoreline.pdf"
+  },
+  "pricing": [
+    {
+      "billing_type": "free"
```
Comment (ytnytn1) [inline shoreline/manifest.json:73]: @mgarabed Thank you for your review! 

Could please help me with the pricing? I would like to add pricing but can't find a specification for it.

File: shoreline/assets/service_checks.json [hunk 3]
```diff
@@ -0,0 +1 @@
+[]
```
Comment (mgarabed) [inline shoreline/assets/service_checks.json:1]: Delete this file

File: shoreline/manifest.json [hunk 4]
```diff
@@ -0,0 +1,75 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "90e1b0ed-0907-4973-929c-7e7f1be0c4f4",
+  "app_id": "shoreline-integration",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Category::Automation",
+    "Category::Marketplace",
```
Comment (mgarabed) [inline shoreline/manifest.json:8]:
```suggestion
```

File: shoreline/manifest.json [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        "media_type": "image",
+        "caption": "Example of fleetwide linux command details",
+        "image_url": "images/fleetwide_linux_command_details.png"
+      }
+    ]
+  },
+  "author": {
+    "homepage": "https://shoreline.io/",
+    "name": "Shoreline Inc",
+    "sales_email": "sales@shoreline.io",
```
Comment (mgarabed) [inline shoreline/manifest.json:45]:
```suggestion
```

File: shoreline/manifest.json [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+      },
+      "metrics": {
+        "prefix": "shoreline_web_service_integration.",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    }
```
Comment (mgarabed) [inline shoreline/manifest.json:65]:
```suggestion
```

File: shoreline/manifest.json [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+  },
+  "legal_terms": {
+    "eula": "assets/terms_of_service_shoreline.pdf"
+  },
+  "pricing": [
+    {
+      "billing_type": "free"
+    }
+  ]
```
Comment (mgarabed) [inline shoreline/manifest.json:74]:
```suggestion
  }
```

File: shoreline/manifest.json (hunk 2 above)
Comment (mgarabed) [reply shoreline/manifest.json:72]: Since this will be on the integrations page, no pricing object needed.

File: shoreline/metadata.csv [hunk 8]
```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```
Comment (mgarabed) [inline shoreline/metadata.csv:1]: Delete this file

**Review by mgarabed — CHANGES_REQUESTED** [review]

**Review by ytnytn1 — COMMENTED** [review]
Hi @mgarabed, I've updated the manifest and removed odd files. Pleaser verify, thanks!

File: shoreline/README.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+
+#### Configure Datadog integration from Shoreline
+
+Log into your Shoreline account and configure your Datadog integration. 
+You will need the API and Application keys of your Datadog account.\
+[Learn more][learn_more] ![link_icon]. 
+
+![integration_example]
+
+## Troubleshooting
```
Comment (nmuesch) [inline shoreline/README.md:43]:
```suggestion
## Support
```

File: shoreline/manifest.json [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Category::Automation",
+    "Offering::UI Extension"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Troubleshooting",
```
[the remark this answers is not in the document]
Comment (ytnytn1) [reply shoreline/manifest.json]: updated

File: shoreline/CHANGELOG.md [hunk 11]
```diff
@@ -0,0 +1 @@
+* Initial setup
```
[the remark this answers is not in the document]
Comment (ytnytn1) [reply shoreline/CHANGELOG.md]: updated

File: shoreline/manifest.json [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+    "Category::Automation",
+    "Offering::UI Extension"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "description": "Build automations to repair triggered monitors",
+    "title": "Shoreline Web Service Integration",
```
Comment (mgarabed) [inline shoreline/manifest.json:16]:
```suggestion
    "title": "Shoreline Remediation",
```

Comment (alexvetras) [reply shoreline/manifest.json:16]: @ytnytn1 "Shoreline UI" or something similar also works, but the current title exceeds the tile's character limit.

File: shoreline/README.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+## Support
+
+For support or requests, please contact Shoreline through the following channels:
+
+Email: [support@shoreline.io][support_email]\
+Documentation is available [here][docs].
+
+[integration_example]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/shoreline/images/integrate_shoreline_and_datadog.png
+[support_email]: support@shoreline.io
```
Comment (anawishnoff) [inline shoreline/README.md:57]:
```suggestion
[2]: support@shoreline.io
```

File: shoreline/README.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+[visit]: https://shoreline.io/datadog?source=DatadogIntTile
```
Comment (anawishnoff) [inline shoreline/README.md:58]:
```suggestion
[3]: https://shoreline.io/datadog?source=DatadogIntTile
```

File: shoreline/README.md [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+[learn_more]: https://docs.shoreline.io/integrations/datadog
+[installation_kubernetis]: https://docs.shoreline.io/installation/kubernetes
+[installation_via_helm]: https://docs.shoreline.io/installation/kubernetes#install-with-helm
```
Comment (anawishnoff) [inline shoreline/README.md:61]:
```suggestion
[6]: https://docs.shoreline.io/installation/kubernetes#install-with-helm
```

File: shoreline/README.md [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+1. Install the Shoreline agent
+2. Configure Datadog integration from Shoreline
+
+
+#### Install the Shoreline agent
+
+Shoreline Agents must be installed on every host you want Shoreline to monitor and act upon. There are a few recommended methods for installing Agents:
+
+1. [Kubernetes][installation_kubernetis] ![link_icon]
+2. [Kubernetes via Helm][installation_via_helm] ![link_icon]
```
Comment (anawishnoff) [inline shoreline/README.md:31]:
```suggestion
2. [Kubernetes via Helm][6] 
```

File: shoreline/README.md [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+You will need the API and Application keys of your Datadog account.\
+[Learn more][learn_more] ![link_icon]. 
+
+![integration_example]
+
+## Support
+
+For support or requests, please contact Shoreline through the following channels:
+
+Email: [support@shoreline.io][support_email]\
```
Comment (anawishnoff) [inline shoreline/README.md:53]:
```suggestion
Email: [support@shoreline.io][2]
```

**Comment by anawishnoff** [comment]
Hi @ytnytn1, I saw that the links were currently broken in your readme so I've added some suggestions to format those correctly - let me know if you see any issues. Thanks!

**Comment by ytnytn1** [comment]
> Hi @ytnytn1, I saw that the links were currently broken in your readme so I've added some suggestions to format those correctly - let me know if you see any issues. Thanks!

Hi @anawishnoff, thank you for your review!  
links look broken because they point to the master branch where this integration doesn't exist yet. I saw a lot of examples here with this approach. I can update links to use relative paths in this case they will work in the current branch also. 
One quick question: is that required to use numbers as the links e.g. `[here][docs]. vs [here][9]`? For me, numbers are less readable

**Comment by anawishnoff** [comment]
Hi @ytnytn1 - apologies, you are correct about that full direct link to the image in the master branch! I've updated my suggestions to reflect that.

As for the numbered list of links, that is a syntax that's required by our documentation team. Thanks!

File: shoreline/README.md [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+3. [Virtual Machines][installation_virtual_machines] ![link_icon]
+
+**About Shoreline Agents**\
+An Agent is an efficient and non-intrusive process running in the background of all your monitored hosts. Agents constantly collect data from both the host and all connected pods and containers. This aggregated data is periodically sent to Shoreline's backend and is used to create Metrics.
+
+Agents are also the secure link between Shoreline and your environment's Resources. Agents can execute actions on your behalf -- everything from simple Linux commands to remediation playbooks. Many Op Language statements pass an API request through Shoreline's backend and onto the relevant Agents, which then execute that command across all targeted Resources.
+
+Since Agents receive commands from Shoreline's backend, they also take automatic remediation steps based on the Alarms, Actions, and Bots you have configured. These objects work in tandem to monitor your fleet and dispatch the appropriate response if something goes wrong.
+
+#### Configure Datadog integration from Shoreline
```
Comment (alai97) [inline shoreline/README.md:41]:
```suggestion
#### Shoreline
```

File: shoreline/README.md [hunk 19]
```diff
@@ -0,0 +1,64 @@
...
+
+Log into your Shoreline account and configure your Datadog integration. 
+You will need the API and Application keys of your Datadog account.\
```
Comment (alai97) [inline shoreline/README.md:44]:
```suggestion
To configure the Datadog-Shoreline integration in Shoreline, you need your Datadog API and Application Keys. 
```

File: shoreline/README.md [hunk 20]
```diff
@@ -0,0 +1,64 @@
...
+[Learn more][learn_more] ![link_icon]. 
+
+![integration_example]
+
+## Support
```
Comment (alai97) [inline shoreline/README.md:49]:
```suggestion
## Troubleshooting
```

File: shoreline/README.md [hunk 21]
```diff
@@ -0,0 +1,64 @@
...
+
+For support or requests, please contact Shoreline through the following channels:
```
Comment (alai97) [inline shoreline/README.md:51]:
```suggestion
Contact Shoreline through email for support and requests.
```

File: shoreline/README.md [hunk 22]
```diff
@@ -0,0 +1,64 @@
...
+Since Agents receive commands from Shoreline's backend, they also take automatic remediation steps based on the Alarms, Actions, and Bots you have configured. These objects work in tandem to monitor your fleet and dispatch the appropriate response if something goes wrong.
+
+#### Configure Datadog integration from Shoreline
+
+Log into your Shoreline account and configure your Datadog integration. 
+You will need the API and Application keys of your Datadog account.\
+[Learn more][learn_more] ![link_icon]. 
+
+![integration_example]
+
```
Comment (alai97) [inline shoreline/README.md:48]:
```suggestion
## Data Collected

### Metrics

The Shoreline integration does not include any metrics.

### Events

The Shoreline integration does not include any events.

### Service Checks

The Shoreline integration does not include any service checks.

```

File: shoreline/manifest.json [hunk 23]
```diff
@@ -0,0 +1,47 @@
...
+  "manifest_version": "2.0.0",
+  "app_uuid": "90e1b0ed-0907-4973-929c-7e7f1be0c4f4",
+  "app_id": "shoreline-integration",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Category::Automation",
+    "Offering::UI Extension"
+  ],
+  "display_on_public_website": true,
+  "tile": {
```
Comment (alai97) [inline shoreline/manifest.json:11]: Are these in place of "public_title", "categories", "type", "is_public", and "integration_id"?

Comment (mgarabed) [reply shoreline/manifest.json:11]: Yes, this is the new Manifest V2 format.

File: shoreline/README.md (hunk 14 above)
Comment (desaialay) [reply shoreline/README.md:58]: @anawishnoff accepted

File: shoreline/README.md [hunk 24]
```diff
@@ -0,0 +1,64 @@
...
+For support or requests, please contact Shoreline through the following channels:
+
+Email: [support@shoreline.io][support_email]\
+Documentation is available [here][docs].
+
+[integration_example]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/shoreline/images/integrate_shoreline_and_datadog.png
+[support_email]: support@shoreline.io
+[visit]: https://shoreline.io/datadog?source=DatadogIntTile
+[learn_more]: https://docs.shoreline.io/integrations/datadog
+[installation_kubernetis]: https://docs.shoreline.io/installation/kubernetes
```
[the remark this answers is not in the document]
Comment (desaialay) [reply shoreline/README.md:60]: @anawishnoff accepted

File: shoreline/README.md [hunk 25]
```diff
@@ -0,0 +1,64 @@
...
+[installation_via_helm]: https://docs.shoreline.io/installation/kubernetes#install-with-helm
+[installation_virtual_machines]: https://docs.shoreline.io/installation/virtual-machines
```
[the remark this answers is not in the document]
Comment (desaialay) [reply shoreline/README.md:62]: @anawishnoff accepted

File: shoreline/README.md (hunk 16 above)
Comment (desaialay) [reply shoreline/README.md:31]: @anawishnoff accepted

File: shoreline/README.md [hunk 26]
```diff
@@ -0,0 +1,64 @@
...
+2. Configure Datadog integration from Shoreline
+
+
+#### Install the Shoreline agent
+
+Shoreline Agents must be installed on every host you want Shoreline to monitor and act upon. There are a few recommended methods for installing Agents:
+
+1. [Kubernetes][installation_kubernetis] ![link_icon]
+2. [Kubernetes via Helm][installation_via_helm] ![link_icon]
+3. [Virtual Machines][installation_virtual_machines] ![link_icon]
```
[the remark this answers is not in the document]
Comment (desaialay) [reply shoreline/README.md:32]: @anawishnoff accepted

File: shoreline/README.md [hunk 27]
```diff
@@ -0,0 +1,64 @@
...
+
+Agents are also the secure link between Shoreline and your environment's Resources. Agents can execute actions on your behalf -- everything from simple Linux commands to remediation playbooks. Many Op Language statements pass an API request through Shoreline's backend and onto the relevant Agents, which then execute that command across all targeted Resources.
+
+Since Agents receive commands from Shoreline's backend, they also take automatic remediation steps based on the Alarms, Actions, and Bots you have configured. These objects work in tandem to monitor your fleet and dispatch the appropriate response if something goes wrong.
+
+#### Configure Datadog integration from Shoreline
+
+Log into your Shoreline account and configure your Datadog integration. 
+You will need the API and Application keys of your Datadog account.\
+[Learn more][learn_more] ![link_icon].
```
[the remark this answers is not in the document]
Comment (desaialay) [reply shoreline/README.md:45]: @anawishnoff accepted

File: shoreline/README.md (hunk 17 above)
Comment (desaialay) [reply shoreline/README.md:53]: @anawishnoff accepted

File: shoreline/README.md [hunk 28]
```diff
@@ -0,0 +1,64 @@
...
+[Learn more][learn_more] ![link_icon]. 
+
+![integration_example]
+
+## Support
+
+For support or requests, please contact Shoreline through the following channels:
+
+Email: [support@shoreline.io][support_email]\
+Documentation is available [here][docs].
```
[the remark this answers is not in the document]
Comment (desaialay) [reply shoreline/README.md:54]: @anawishnoff accepted

File: shoreline/README.md [hunk 29]
```diff
@@ -0,0 +1,64 @@
...
+
+Shoreline incident automation enables DevOps and Site reliability engineers (SREs) to interactively **debug at scale** and quickly **build remediations** to eliminate repetitive work.
+
+**The debug and repair feature** allows engineers to execute commands in real-time, across their server farm without having to SSH into the servers individually. You can execute anything that can be typed at the Linux command prompt, including Linux commands, shell scripts, calls to their cloud provider APIs, etc. Then, quickly turn these debug sessions into automations connected to Datadog monitors. The Shoreline app will automatically execute that automation whenever the monitor is triggered. This greatly reduces Mean Time To Repair (MTTR) and manual work.
+
+Shoreline helps everyone on call to be as good as your best SRE.  Shoreline
+arms your entire on-call team with best practice debugging tools and
+approved remediation actions.  This helps the whole team find and fix
+incidents more quickly with fewer escalations.  It also ensures that incidents
+are fixed correctly the first time with fewer mistakes.
```
[the remark this answers is not in the document]
Comment (desaialay) [reply shoreline/README.md:13]: @alai97 We would like to keep line #9. "Shoreline helps everyone on call to be as good as your best SRE."

Other suggested changes are accepted. Also reverting recommended "less" to "fewer". 

Final paragraph will be:

Shoreline helps everyone on call to be as good as your best SRE. Shoreline arms your on-call team with debugging tools and approved remediation actions, helping you fix incidents faster with fewer escalations and ensuring that incidents are fixed correctly the first time with fewer mistakes.

File: shoreline/README.md [hunk 30]
```diff
@@ -0,0 +1,64 @@
...
+
+To get started, [visit][visit] ![link_icon] to set up a trial account.
+## Setup
+
+### Installation
+
+Follow the steps below to configure the integration:
+
+1. Install the Shoreline agent
+2. Configure Datadog integration from Shoreline
```
[the remark this answers is not in the document]
Comment (desaialay) [reply shoreline/README.md:23]: @alai97 accepted.

File: shoreline/README.md [hunk 31]
```diff
@@ -0,0 +1,64 @@
...
+1. [Kubernetes][installation_kubernetis] ![link_icon]
+2. [Kubernetes via Helm][installation_via_helm] ![link_icon]
+3. [Virtual Machines][installation_virtual_machines] ![link_icon]
+
+**About Shoreline Agents**\
+An Agent is an efficient and non-intrusive process running in the background of all your monitored hosts. Agents constantly collect data from both the host and all connected pods and containers. This aggregated data is periodically sent to Shoreline's backend and is used to create Metrics.
+
+Agents are also the secure link between Shoreline and your environment's Resources. Agents can execute actions on your behalf -- everything from simple Linux commands to remediation playbooks. Many Op Language statements pass an API request through Shoreline's backend and onto the relevant Agents, which then execute that command across all targeted Resources.
+
+Since Agents receive commands from Shoreline's backend, they also take automatic remediation steps based on the Alarms, Actions, and Bots you have configured. These objects work in tandem to monitor your fleet and dispatch the appropriate response if something goes wrong.
```
[the remark this answers is not in the document]
Comment (desaialay) [reply shoreline/README.md:39]: @alai97 accepted.

File: shoreline/README.md (hunk 18 above)
Comment (desaialay) [reply shoreline/README.md:41]: @alai97 accepted.

**Comment by desaialay** [comment]
@anawishnoff @alai97 I have accepted nearly all the suggestions. I could not make the changes because I do not have the permissions (not sure why but I need to check). @ytnytn1 will update.

@alai97 I have requested clarification about "events" not included. Please advise. 

Thanks.

File: shoreline/README.md (hunk 29 above)
[the remark this answers is not in the document]
Comment (alai97) [reply shoreline/README.md:13]:
```suggestion
Shoreline helps everyone on call be as good as your best SRE. Shoreline arms your on-call team with debugging tools and approved remediation actions, helping you fix incidents faster with less escalations and ensuring that incidents are fixed correctly the first time with less mistakes.
```

[the remark this answers is not in the document]
Comment (alai97) [reply shoreline/README.md:13]: Works for me, @desaialay!

File: shoreline/README.md (hunk 22 above)
Comment (alai97) [reply shoreline/README.md:48]: Hey @desaialay, from an end user's perspective, the integration does not provide any events for them to see in Datadog, so we add this boilerplate text for our integrations 🙃

File: shoreline/README.md (hunk 30 above)
[the remark this answers is not in the document]
Comment (ytnytn1) [reply shoreline/README.md]: @alai97 @desaialay 
This change confuses me a bit. It says I need to install the integration with Datadog Shoreline integration tile.  And the link leads to https://app.datadoghq.com/account/settings#integrations/shoreline but I'm already located in the integration tile. Why should it lead to the same location where I'm already located?

**Pull request merged**